examples/pxScene2d/src/CMakeLists.txt

```diff
@@ -287,6 +287,21 @@ else ()
     add_definitions(-DPX_SCENE_VERSION=${PXSCENE_VERSION})
 endif("${PXSCENE_VERSION}" STREQUAL "")
 
+execute_process(
+  COMMAND git describe --tags --always --long --dirty
```

Comment:
This is a great idea but surprisingly against your https://github.com/pxscene/pxCore/issues/790 request.
Please note that for #790 a bare tarball is necessary and this will be no longer Git repository _(.git sub directory)_. So using the propsed solution we will end up in 0.0.0.0-0-0000000000-dirty version all the time.

Reading a version from a file (which can be included in a tarball) might be a solution which will work in both cases.

Reply:
I like the idea of generating a version.txt at configuration / compile time, possibly from the `git describe` if it exists.

About the tarball comment, what is that required for?  It doesn't immediately seem necessary from when I was looking at CPack.

Reply:
>About the tarball comment, what is that required for?

If any Linux distribution will be interested in providing native package e.g. .rpm or .deb the only thing they really need is a tarball from https://github.com/pxscene/pxCore/releases. Unfortunately executing git command within this unpacked tarball will simply not return the proper software version.